app/models/mixins/event_mixin.rb

```diff
@@ -55,11 +56,9 @@ def miq_event_filter
     filter
   end
 
-  private
-
```

Comment:
This is correct. But do we want to leave the private in here anyway?

Or I can put a comment onto `find_one_event` to say it is private